src/LaunchDarkly/LDClient.php

```diff
@@ -1,31 +1,40 @@
 <?php
 namespace LaunchDarkly;
 
-use Exception;
+use Monolog\Handler\ErrorLogHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
 
 /**
  * A client for the LaunchDarkly API.
  */
 class LDClient {
     const DEFAULT_BASE_URI = 'https://app.launchdarkly.com';
-    const VERSION = '1.0.0';
+    const VERSION = '2.0.0';
 
+    /** @var string */
     protected $_apiKey;
+    /** @var string */
     protected $_baseUri;
-    protected $_client;
+    /** @var EventProcessor */
     protected $_eventProcessor;
-    protected $_offline;
-    protected $_events = true;
+    /** @var  bool */
+    protected $_offline = false;
+    /** @var bool */
+    protected $_send_events = true;
+    /** @var array|mixed */
     protected $_defaults = array();
+    /** @var mixed|LoggerInterface */
+    protected $_logger;
 
     /** @var  FeatureRequester */
     protected $_featureRequester;
 
     /**
      * Creates a new client instance that connects to LaunchDarkly.
      *
-     * @param string $apiKey  The API key for your account
-     * @param array  $options Client configuration settings
+     * @param string $apiKey The API key for your account
+     * @param array $options Client configuration settings
```

Comment:
I think the documentation for options needs to be updated for (`send_events`, `offline`, `logger`)